### DISCUSSION TITLE
Dealing with partial transparency when making gifs

### DISCUSSION THREAD

**Discussion by Jsfasdf250** [ImageMagick/ImageMagick#6424] [Digital Image Processing]
None of the following methods are allowed:
- Using a different file format (APNG and TIFF aren't as widely supported, especially with larger images)
- Dithering (It makes it look bad when zoomed in or out)
- Adding a solid background to the partially transparent parts (It would look bad when a different background color is used)

**Comment by fmw42** [comment]
GIFs do not support partial transparency. It only use binary transparency (pure transparent and pure opaque).

**Comment by Jsfasdf250** [comment]
Anyway, is dithering always really that bad?

**Reply by fmw42** [reply]
You have not shown us any of your output attempts!

**Comment by snibgo** [comment]
Try it and see. It might be acceptable to you, for your purposes.

**Reply by Jsfasdf250** [reply]
I made many gifs with dithering, though many people complained about how the gifs look when looking at them close up.

**Reply by fmw42** [reply]
Define bad!  You have not shown us any of your output attempts!
